v9.2 DMA2D build issue

### LVGL version

9.2 84b28ff688b9b8b33682191c37554194d5ae1950

### What happened?

Similar to the issues with PXP I am facing build errors due to moval of certain structs to private headers it seems:
```
zephyr-workspace/modules/lib/gui/lvgl/src/draw/dma2d/lv_draw_dma2d_fill.c: In function 'lv_draw_dma2d_opaque_fill':
zephyr-workspace/modules/lib/gui/lvgl/src/draw/dma2d/lv_draw_dma2d_fill.c:39:5: error: unknown type name 'lv_draw_fill_dsc_t'; did you mean 'lv_draw_glyph_dsc_t'?
   39 |     lv_draw_fill_dsc_t * dsc = u->task_act->draw_dsc;
      |     ^~~~~~~~~~~~~~~~~~
      |     lv_draw_glyph_dsc_t
zephyr-workspace/modules/lib/gui/lvgl/src/draw/dma2d/lv_draw_dma2d_fill.c:40:31: error: request for member 'base' in something not a structure or union
   40 |     lv_color_format_t cf = dsc->base.layer->color_format;
      |                               ^~
ephyr-workspace/modules/lib/gui/lvgl/src/draw/dma2d/lv_draw_dma2d_fill.c:44:82: error: request for member 'color' in something not a structure or union
   44 |     uint32_t reg_to_mem_color = lv_draw_dma2d_color_to_dma2d_ocolr(output_cf, dsc->color);
      |                                                                                  ^~
zephyr-workspace/modules/lib/gui/lvgl/src/draw/dma2d/lv_draw_dma2d_fill.c: In function 'lv_draw_dma2d_fill':
zephyr-workspace/modules/lib/gui/lvgl/src/draw/dma2d/lv_draw_dma2d_fill.c:72:5: error: unknown type name 'lv_draw_fill_dsc_t'; did you mean 'lv_draw_glyph_dsc_t'?
   72 |     lv_draw_fill_dsc_t * dsc = u->task_act->draw_dsc;
      |     ^~~~~~~~~~~~~~~~~~
      |     lv_draw_glyph_dsc_t
zephyr-workspace/modules/lib/gui/lvgl/src/draw/dma2d/lv_draw_dma2d_fill.c:73:27: error: request for member 'color' in something not a structure or union
   73 |     lv_color_t color = dsc->color;
      |                           ^~
zephyr-workspace/modules/lib/gui/lvgl/src/draw/dma2d/lv_draw_dma2d_fill.c:74:31: error: request for member 'base' in something not a structure or union
   74 |     lv_color_format_t cf = dsc->base.layer->color_format;
      |                               ^~
zephyr-workspace/modules/lib/gui/lvgl/src/draw/dma2d/lv_draw_dma2d_fill.c:75:23: error: request for member 'opa' in something not a structure or union
   75 |     lv_opa_t opa = dsc->opa;
      |                       ^~
[52/322] Building C object modules/lvgl/CMakeFiles/modules__lvgl.dir zephyr-workspace/modules/lib/gui/lvgl/src/draw/sw/blend/lv_draw_sw_blend_to_rgb565.c.obj
ninja: build stopped: subcommand failed.
FATAL ERROR: command exited with status 1: /usr/bin/cmake --build zephyr-workspace/zephyr/build
```

Revision 84b28ff688b9b8b33682191c37554194d5ae1950

### How to reproduce?

_No response_

--- a/src/draw/dma2d/lv_draw_dma2d_private.h
+++ b/src/draw/dma2d/lv_draw_dma2d_private.h
@@ -18,6 +18,7 @@ extern "C" {
 #if LV_USE_DRAW_DMA2D
 
 #include "../lv_draw_private.h"
+#include "../sw/lv_draw_sw.h"
 #include LV_DRAW_DMA2D_HAL_INCLUDE
 
 /*********************

